BaseFilter/sections/specific.txt

```diff
@@ -326,9 +326,10 @@ lovethemaldives.com##div[class*="-rectangle-"]
 allnovascotia.com##div[style="height:130px;text-align:center;background-color:#eef2f5"]
 tutlehd.xyz###overlayer
 portableapps.com##.ez-sidebar-wall
-check-host.net#?#div + div[class*=" "]:last-child:contains(/^(×|✕|✖|X|☓|χ|x|⨯)$/):upward(2)
-check-host.net###hostign
-/^https:\/\/check-host\.net\/images\/[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net#?#div + div[class*=" "]:last-child:contains(/^(χ|Χ|х|𝕏|🗙|🗶|🗴|✘|✗|⨯|☓|×|✕|✖|X|x)$/):upward(2)
+check-host.net###hostigm
+/^https:\/\/check-host\.net\/images\/[a-z]+[0-9]?_?[0-9]+[a-z]+(_|[0-9]?[a-z-])+(s?ale|en|ru)?\.png\?[0-9]+$/$domain=check-host.net
+check-host.net##a[href^="https://specialrdp.com/"]
```

Comment:
Not needed as you have `check-host.net###hostigm`.

Reply:
It was backup rule do to them always updating the site